bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_rdataframe.py

```diff
@@ -74,16 +88,77 @@ def RDataFrameAsNumpy(df, columns=None, exclude=None):
     return py_arrays
 
 
+def _histo_profile(self, fixed_args, *args):
+    # Check wheter the user called one of the HistoXD or ProfileXD methods
+    # of RDataFrame with a tuple as first argument; in that case,
+    # extract the tuple items to construct a model object and call the
+    # original implementation of the method with that object.
+
+    # Parameters:
+    # self: instantiation of RDataFrame
+    # fixed_args: list containing e.g. '_OriginalHisto1D' and
```

Comment:
`fixed_args` is a tuple, not a list, and again Histo1D is not a method, it is a class.
Proposal: `fixed_args: tuple containing the original name of the method being pythonised and the class of the model object to construct.`